2021/OffSeason2021/src/main/include/commands/AutoPathSequence.h

```diff
@@ -10,9 +10,6 @@
 
 #pragma once
 
-#include <spdlog/fmt/ostr.h>
-#include <spdlog/spdlog.h>
-
```

Comment:
It has been so long since I was deep in the C++ ecosystem - I forgot that these #includes weren't free.  Good call here!

Reply:
We were particularly fighting the school laptop 12 which we have been using consistently for months. The past week it jumped to build times around 8 minutes or more and inconsistent times. This is an attempt to reduce the build times, but the results are inconclusive due to how many builds we've been able to try. It seems to help so far. Entirely clean build is about 8 minutes (includes building the spdlog library and pulling in vendor libraries), making an entirely trivial change ~1m. Somewhere in between ends up at about 4 minutes. And for the record, laptop 12 is a 5-6 year old i7 with 8GB of memory (which is probably 75% utilized).